app/lib/check_notification.rb

```diff
@@ -1,16 +1,13 @@
 module CheckNotification
   class InfoCodes
     SENT_TO_TRASH_BY_RULE = 1
-    MOVED_TO_PROJECT_BY_RULE = 2
     BANNED_SUBMITTER_BY_RULE = 3
-    COPIED_TO_PROJECT_BY_RULE = 4
```

Comment:
These are actually related to the `Project` removal, right

Reply:
right